include/ppx/application.h

```diff
@@ -232,6 +233,8 @@ struct ApplicationSettings
     bool        headless              = false;
     bool        enableImGui           = false;
     bool        allowThirdPartyAssets = false;
+    // Set to true to opt-in for metrics
```

Comment:
nit: "." at the end
also, clang format stopped aligning variables?

Reply:
Fixed.
And yes, clang-format stops aligning after the comment.